### PR TITLE
chore(ci) publish bundle and push back to main

### PR DESCRIPTION
**Pull request by HenriPro**
Publishes the the swagger-ui-bundle.js to npm and github package, as well as adds action to commit back to main

### CODE REVIEW

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -25,6 +25,19 @@ jobs:
...
+        with:
+          persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal access token.
+          fetch-depth: 0 # otherwise, there would be errors pushing refs to the destination repository.
+        run: |
+          date > generated.txt
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git add .
+          git commit -m "publish new dist 🎉 "
+          git push}}
```
Comment (zekth) [inline .github/workflows/npm-publish.yml:39]: Typo

**Review by zekth — CHANGES_REQUESTED** [review]

File: .github/workflows/npm-publish.yml [hunk 2]
```diff
@@ -25,6 +25,19 @@ jobs:
           # A file, directory or wildcard pattern that describes what to upload
           path: dist/*
           if-no-files-found: error
+      - name: Commit files
+      - uses: actions/checkout@v2
```
Comment (zekth) [inline .github/workflows/npm-publish.yml:29]:
```suggestion
        uses: actions/checkout@v2
```

**Review by zekth — CHANGES_REQUESTED** [review]

File: .github/workflows/npm-publish.yml [hunk 3]
```diff
@@ -25,6 +25,19 @@ jobs:
...
+          persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal access token.
+          fetch-depth: 0 # otherwise, there would be errors pushing refs to the destination repository.
+        run: |
+          date > generated.txt
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git add .
+          git commit -m "publish new dist 🎉 "
+          git push
+          branch: ${{ github.ref }}
```
Comment (Mierenga) [inline .github/workflows/npm-publish.yml:40]: Is this supposed to be part of the run script or another property? I see the checkout actions doc has `ref` but not `branch`: https://github.com/actions/checkout

Comment (HenriPro) [reply .github/workflows/npm-publish.yml:40]: I think this is not needed good 👁️

Comment (HenriPro) [reply .github/workflows/npm-publish.yml:40]: I was going to use a different action before I realized checkout can do this

**Review by Mierenga — APPROVED** [review]
Looks good to test out on the next publish!

**Pull request merged**